indexer/osm_editor.cpp

```diff
@@ -0,0 +1,109 @@
+#include "osm_editor.hpp"
+
+#include "indexer/feature_decl.hpp"
+#include "indexer/index.hpp"
+
+#include "std/map.hpp"
+#include "std/set.hpp"
+
+namespace osm
```

Comment:
Not good, as there is a big dependencies issue. Editor works with features. Indexer works with editor. And we are wasting our time on unimportant and irrelevant stuff.

> On 18.11.2015, at 23.58, ygorshenin <notifications@github.com> wrote:
> 
> In indexer/osm_editor.cpp:
> 
> > @@ -0,0 +1,109 @@
> > +#include "osm_editor.hpp"
> > +
> > +#include "indexer/feature_decl.hpp"
> > +#include "indexer/index.hpp"
> > +
> > +#include "std/map.hpp"
> > +#include "std/set.hpp"
> > +
> > +namespace osm
> And, if at some point in the future we will decide to move opening_hours from 3party, that "osm" library will be a good place for that.
> 
> —
> Reply to this email directly or view it on GitHub.
>